lib/addons/xray-adot-addon/index.ts

```diff
@@ -0,0 +1,86 @@
+import { ClusterAddOn, ClusterInfo, Values } from "../../spi";
+import { dependable, loadYaml, readYamlDocument } from "../../utils";
+import { AdotCollectorAddOn } from "../adot";
+import { Construct } from 'constructs';
+import { KubectlProvider, ManifestDeployment } from "../helm-addon/kubectl-provider";
+
+/**
```

Comment:
Minor: this should be the doc over the add-on class, then customers will see it when they use the add-on in the IDE.